package.json

```diff
@@ -59,11 +68,6 @@
     "url": "https://github.com/wechaty/plugin-contrib/issues"
   },
   "homepage": "https://github.com/wechaty/plugin-contrib#readme",
-  "git": {
```

Comment:
Why do you remove the git hook?

Reply:
This is an error because I deleted it during submission and forgot to restore it afterwards.